internal/pgbackrest/pgbackrest_settings.go

```diff
@@ -0,0 +1,147 @@
+package pgbackrest
+
+import (
+	"encoding/json"
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+	"gopkg.in/ini.v1"
+)
+
+const (
+	BackupPath        = "backup"
+	BackupInfoIni     = "backup.info"
+	BackupManifestIni = "backup.manifest"
+
+	BackupFolderName    = "backup"
+	BackupDataDirectory = "pg_data"
+)
+
+type BackupSettings struct {
```

Comment:
What is the difference between `BackupSettings` and `PgbackrestBackupSettings`?

Reply:
Not so much differences, remove BackupSettings and use PgbackrestBackupSettings instead